docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -55,12 +56,35 @@
 \wss{You are free to modify this template.}
 
 \section{Introduction}
-
-\wss{You can include your definition of what a hazard is here.}
+This document outlines the hazard analysis of Sayyara. The definition of hazard is any property or
+condition in the system along with conditions in the environment that may cause harm or damage.
+This definition is from Nancy Leveson's work. The hazards for Sayyara include security and usage
+hazards such as protecting personal information, database failures, and having no internet
+connection.
 
 \section{Scope and Purpose of Hazard Analysis}
+The scope of the hazard analysis is to identify any hazards that may arise when using the
+application, their causes, coming up with steps to eliminate or mitigate the effect of the hazard.
+The purpose of the hazard analysis is to pinpoint areas where hazards may arise and their effects
+and come up with mitigation steps. Through completing the hazard analysis, safety and security
+requirements will be developed early in the design process to minimize the risk of having hazards
+occur without plans in place to reduce or mitigate the effects.
 
 \section{System Boundaries and Components}
+The system consists of:
+\begin{enumerate}
+	\item The application's front-end and back-end components in the major categories:
```

Comment:
Do we say front-end or frontend (same applies to back-end)? I think we have been switching between the two across report. I think we should be consistent.